corehq/apps/integration/models.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+from memoized import memoized
+from corehq import toggles
+
+from django.db import models
+
+
+class SimprintsIntegration(models.Model):
+    domain = models.CharField(max_length=128, unique=True)
+    is_enabled = models.BooleanField(default=False)
+    project_id = models.CharField(max_length=255)
+    user_id = models.CharField(max_length=255)
+    module_id = models.CharField(max_length=255)
```

Comment:
@djmore9 / @ctsims Should we support a project having multiple module ids?

@biyeun How challenging would it be in the future to migrate this to a dict, likely HQ module id => Simprints module id?

Reply:
Simprints has mixed messages on the module Id.

My guess is that the best option isn't a dictionary, but actually a per-user sync field, because I think they  would expect to use that to be able to split up data for different users

Reply:
I think when we get to that stage we can figure out a plan. I can't imagine a migration to something different being too much more of a heavy lift, but I want to keep things as simple as possible for the first iteration rather than tread into over-engineering territory for a situation that may or may not arise...

I also don't have great context on what the simprints `module_id` is for (which is what this refers to, not HQ).

Reply:
in case there was any confusion, all the values here are to be supplied from simprints, not HQ.